msteams-platform/concepts/messaging-extensions/create-extensions.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Action based messaging extensions
+description: Describes how to develop action based messaging extensions
```

Comment:
Please change the description to match the first para of the doc (or an abbreviated version of it).

Reply:
description: Create action based messaging extensions to allow users to trigger 3rd party services

Reply:
Changed 3rd party per comment below.